### PR TITLE
Fixed Interactive dialog radio buttons style

### PR DESCRIPTION
**Pull request by nixusUM**
#### Summary
When build interactive dialog radio elements not shown when device use light theme(Android) and difficult to read(iPhone)

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/22326

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: TECNO P8 Android, Samsung Galaxy S7 Edge Android, iPhone 11 Pro

#### Screenshots
before: 

image

after:

image

#### Release Note
```release-note
Fixed Interactive dialog radio buttons style when light theme use
```

### CODE REVIEW

File: app/components/settings/radio_setting/radio_entry.tsx [hunk 1]
```diff
@@ -31,6 +31,10 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
             fontSize: 12,
             color: theme.linkColor,
         },
+        text: {          
+            fontSize: 12,  
+            color: theme.centerChannelColor,        
+        },
```
Comment (enahum) [inline app/components/settings/radio_setting/radio_entry.tsx:37]:
```suggestion
        text: {
            fontSize: 12,
            color: theme.centerChannelColor,
        },
```

**Review by enahum — CHANGES_REQUESTED** [review]

File: app/components/settings/radio_setting/radio_entry.tsx (hunk 1 above)
Comment (nixusUM) [reply app/components/settings/radio_setting/radio_entry.tsx]: fixed

**Comment by nixusUM** [comment]
> /check-cla

**Comment by enahum** [comment]
Hi @nixusUM you have some linting issues, can you address them please?

**Comment by nixusUM** [comment]
> Hi @nixusUM you have some linting issues, can you address them please?

hi, I fixed linting issues, but build failed again and I don't understand what need to do now

**Comment by nixusUM** [comment]
@enahum Hi, any update?

**Review by enahum — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Review by asaadmahmood — APPROVED** [review]
LGTM, though the checkmark seems small, but I know that's not part of this PR.

**Pull request merged**